src/shippingservice/main.go

```diff
@@ -120,14 +120,8 @@ func (s *server) GetQuote(ctx context.Context, in *pb.GetQuoteRequest) (*pb.GetQ
 	log.Info("[GetQuote] received request")
 	defer log.Info("[GetQuote] completed request")
 
-	// 1. Our quote system requires the total number of items to be shipped.
-	count := 0
-	for _, item := range in.Items {
-		count += int(item.Quantity)
-	}
-
-	// 2. Generate a quote based on the total number of items to be shipped.
-	quote := CreateQuoteFromCount(count)
+	// 1. Generate a quote based on the total number of items to be shipped.
+	quote := CreateQuoteFromCount(0)
 
 	// 3. Generate a response.
```

Comment:
`nit:` can we re adjust this comment to follow the number order `1. 2. ...`

Reply:
Good catch!